etc/config/c++.amazon.properties

```diff
@@ -3697,12 +3697,40 @@ libs.xtl.versions.053.path=/opt/compiler-explorer/libs/xtl/0.5.3/include
 libs.xtl.versions.0416.version=0.4.16
 libs.xtl.versions.0416.path=/opt/compiler-explorer/libs/xtl/0.4.16/include
 
+libs.ztdcuneicode.name=ztd.cuneicode
+libs.ztdcuneicode.url=https://github.com/soasis/cuneicode
+libs.ztdcuneicode.description=Robust C-style text encoding library<br /><br />Note that to use, you must add the "ztd.idk" library
```

Comment:
`libs.ztdcunicode.staticliblink=ztd.cuneicode`

Reply:
Got it!